docs/src/prototypes/chatPane/chatPaneHeader.tsx

```diff
@@ -7,6 +7,17 @@ export interface ChatPaneHeaderProps {
   chat?: ChatData
 }
 
+const screenReaderMessageContainerStyles: React.CSSProperties = {
```

Comment:
you're defining and using identical `screenReaderMessageContainerStyles` in 3 different files.
Would be better to have it in one place and export there (for e.g. create `styles.ts` and put there), then import these styles where you need.

And maybe to rename it `screenReaderMessageContainerStyles` -> `screenReaderContainerStyles`, as it is used not only for messages.